src/internal/Subscriber.ts

```diff
@@ -135,66 +121,46 @@ export class Subscriber<T> extends Subscription implements Observer<T> {
   }
 }
 
-/**
- * This bind is captured here because we want to be able to have
- * compatibility with monoid libraries that tend to use a method named
- * `bind`. In particular, a library called Monio requires this.
- */
-const _bind = Function.prototype.bind;
+class ConsumerObserver<T> implements Observer<T> {
+  constructor(private partialObserver: Partial<Observer<T>>) {}
 
-function bind<Fn extends (...args: any[]) => any>(fn: Fn, thisArg: any): Fn {
-  return _bind.call(fn, thisArg);
-}
-
-export class SafeSubscriber<T> extends Subscriber<T> {
-  constructor(observerOrNext?: Partial<Observer<T>> | ((value: T) => void) | null) {
-    super();
+  next(value: T): void {
+    if (this.partialObserver.next) {
+      try {
+        this.partialObserver.next(value);
+      } catch (error) {
+        reportUnhandledError(error);
```

Comment:
Note the difference here because we don't have to worry about deprecated error behavior.